vignettes/concepts_conventions.Rmd

```diff
@@ -0,0 +1,285 @@
+---
+title: "Programming Concepts and Conventions"
```

Comment:
What about this going in Advanced User Guides?.  I feel like after today's discussion we don't want to scare people away!!

Reply:
i'd rather not hide it more  (by p[lacing it further down) if at all possible. most of the material in this vignette is things that users do have to consider even if they are not hugely proficient in admiral yet.

Reply:
I wonder if we should restructure the "User Guides" menu. I think especially the "Advanced User Guides" section title is not helpful. What does "advanced" mean? For me it sounds like "this information is not required for the daily admiral usage". However, I would recommend reading "Programming Concepts and Conventions", "Date and Time Imputation", "General Derivations", and "Higher Order Functions" to everyone who is using admiral in a study.
What about the following menu?
### General Guides
- Programming Concepts and Conventions
- General Derivations
- Higher Order Functions
### ADaM Script Guides
- Creating an ADSL
- Creating an OCCDS ADaM
- Creating a BDS Findings ADAM
- Creating a BDS Exposure ADaM
- Creating a BDS Time-to-Event ADaM
- Creating Questionnaire ADaMs
- Creating a PK NCA or Population PK ADaM
- Creating a Hy's Law ADaM
### Special Topics Guides
- Date and Time Imputation
- Visit and Period Variables
- Queries Dataset Documentation
- Lab Grading
### FAQ
- FAQ

Reply:
oooooohhhh this is lovely!!!  Do you mind restructuring this @manciniedoardo ?

Reply:
done!